corehq/apps/users/cases.py

```diff
@@ -8,8 +8,10 @@
 def user_db():
     return CouchUser.get_db()
 
+
 def get_owner_id(case):
-    return case.owner_id if case.owner_id is not None else case.user_id
+    return case.owner_id or case.user_id
```

Comment:
issue was that clean restore is [using this function for liveness checks](https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/phone/data_providers/case/clean_owners.py#L150) but [the view](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqcase/_design/views/by_owner/map.js#L3) and [the serializer](https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/case/xml/generator.py#L144) were just using `or` instead of checking `None` explicitly